scripts/install.sh

```diff
@@ -45,11 +51,12 @@ done
 
 # Parse short options
 OPTIND=1
-while getopts "n:dbh" opt
+while getopts "n:t:ebh" opt
 do
    case "$opt" in
       n ) network="$OPTARG" ;;
-      d ) skip_deployment=${OPTARG:-true} ;;
+      t ) tbtc_path="$OPTARG" ;;
```

Comment:
We may want a similiar option for `threshold-network/solidity-contracts`.

Reply:
[0bf29c2](https://github.com/keep-network/keep-core/pull/3096/commits/0bf29c287f2f9ba6de4514d753b718a8815baf97)